### PR TITLE
Reorganize portal profile options

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

With more options being push into the portal profile.
We should start organizing the portal profile options into different tabs
## Impacts

This impact the creation and modification of portal profiles and the viewing of Profile files
## Delete branch after merge

NO

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
I added screenshots of the new changes

screen shot 2015-01-15 at 10 57 57 am

screen shot 2015-01-15 at 10 56 03 am
screen shot 2015-01-15 at 10 56 42 am

**Comment by jrouzierinverse** [comment]
Please Review

File: html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm [hunk 1]
```diff
@@ -61,7 +86,6 @@ has_field 'logo' =>
   (
    type => 'Text',
    label => 'Logo',
-   required => 1,
```
Comment (dwlfrth) [inline html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm:64]: Finally ! Ahaha
Will it takes the default one if not configured ?

**Comment by cgx** [comment]
I would suggest to keep _Enable billing engine_ with the authentication sources.

**Comment by dwlfrth** [comment]
After discussion with Francis, even if I was the one asking to move the 'Enable billing engine', I agree with him.

File: html/pfappserver/root/config/profile/view.tt [hunk 2]
```diff
@@ -29,19 +29,25 @@
...
+        </li>
         <li>
           <a href="[% c.pf_hash_for(c.controller('Config::Profile').action_for('files'),[id]) %]" >[% l('Files')  %]</a>
         </li>
       </ul>
       <div class="tab-pane fade in [% 'active' IF tab == 'settings' %]" id="settings">
       [% INCLUDE 'config/profile/tab-content.tt' tab='settings' %]
+      </div>
+      <div class="tab-pane fade in [% 'active' IF tab == 'captive-portal' %]" id="captive-portal">
+      [% INCLUDE 'config/profile/captive-portal-settings.tt' tab='settings' %]
```
Comment (dwlfrth) [inline html/pfappserver/root/config/profile/view.tt]: Indentation inside a div

File: html/pfappserver/lib/pfappserver/Form/Config/Profile/Default.pm [hunk 3]
```diff
@@ -16,21 +16,11 @@ use HTML::FormHandler::Moose;
...
-=cut
-
-has_block 'definition' =>
-  (
-   render_list => [ qw(id description logo redirecturl always_use_redirecturl reuse_dot1x_credentials billing_engine nbregpages) ],
-  );
 
 has_field '+redirecturl' => ( required => 1 );
 
+has_field '+logo' => ( required => 1 );
```
Comment (dwlfrth) [inline html/pfappserver/lib/pfappserver/Form/Config/Profile/Default.pm]: Dang! Should not be required...

File: html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm [hunk 4]
```diff
@@ -23,6 +23,31 @@ use pf::ConfigStore::Provisioning;
...
+  (
+   render_list => [ qw(id description reuse_dot1x_credentials  ) ],
+  );
+
+=head2 captive_portal
+
+The captival portal block
+
+=cut
+
```
Comment (dwlfrth) [inline html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm:44]: Remove one blank line...

File: html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm (hunk 1 above)
Comment (jrouzierinverse) [inline html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm:64]: Yes

File: html/pfappserver/root/config/profile/create.tt [hunk 5]
```diff
@@ -28,19 +28,25 @@
...
+        </li>
         <li class="disabled">
           <a>[% l('Files')  %]</a>
         </li>
       </ul>
       <div class="tab-pane fade in active" id="settings">
       [% INCLUDE 'config/profile/tab-content.tt' tab='settings' %]
+      </div>
+      <div class="tab-pane fade in [% 'active' IF tab == 'captive-portal' %]" id="captive-portal">
+      [% INCLUDE 'config/profile/captive-portal-settings.tt' tab='settings' %]
```
Comment (dwlfrth) [inline html/pfappserver/root/config/profile/create.tt]: Indentation inside a div

**Comment by jrouzierinverse** [comment]
Please Review

**Comment by dwlfrth** [comment]
Please review but old comments are still there.
There's one blank line to be removed and a conversation within me and Francis talking about taking back "Billing engine" in the first tab...
Will review once changes will be completed

**Comment by jrouzierinverse** [comment]
Which old comment are you referring in this pull request?
I do not see any comments here referring to it.
The Billing engine tab is now moved back to the first tab.
The empty space was removed in a previous commit.

**Comment by dwlfrth** [comment]
Rebase and merge me please.

**Comment by jrouzierinverse** [comment]
Done

**Pull request merged**